rescript/get_gtdb.py

```diff
@@ -10,135 +10,141 @@
 import tempfile
 import tarfile
 import warnings
-
-import qiime2
+import pandas as pd
 from urllib.request import urlretrieve
 from collections import defaultdict
 from urllib.error import HTTPError
-from rescript.get_data import _gzip_decompress
+
+from q2_types.feature_data import (TSVTaxonomyFormat, DNAFASTAFormat,
+                                   DNAIterator)
 
 # Different versions may have different file names for archaea and
 # bacteria. for example 'ar53' and 'bac120' mean that the GTDB phylogeny
 # is based on 53 and 120 concatenated proteins (cp), respectively.
 # If this changes we can set up a conditional statemnt below.
-VERSION_MAP_DICT = {'214': {'Archaea': 'ar53', 'Bacteria': 'bac120'},
-                    '207': {'Archaea': 'ar53', 'Bacteria': 'bac120'},
-                    '202': {'Archaea': 'ar122', 'Bacteria': 'bac120'}}
-
+VERSION_MAP_DICT = {'214.1': {'Archaea': 'ar53', 'Bacteria': 'bac120'},
```

Comment:
I was pondering this myself. The issue is that the http paths all contain a folder with a decimal:
`https://data.gtdb.ecogenomic.org/releases/releases/release207/202.0/`
`https://data.gtdb.ecogenomic.org/releases/releases/release207/207.0/`
`https://data.gtdb.ecogenomic.org/releases/releases/release207/214.0/`
`https://data.gtdb.ecogenomic.org/releases/releases/release207/214.1/`

At the time I figured I'd just use that folder label and then parse out the base version, *e.g.* 214 from 214.1 rather than split off the decimal and rebuild it for the http string. 

I can do it, I think all I'd need to do is check for a decimal for `version` and then use that for the folder, then append a `.0` to those versions that do not have a `0.1` etc...

Reply:
ah makes sense... let's use the decimals then.

Reply:
The PR should be good to go then. :-)